packages/tools/devtools/devtools-core/src/test/DefaultVisualizers.spec.ts

```diff
@@ -399,32 +399,94 @@ describe("DefaultVisualizers unit tests", () => {
 			"test",
 		);
 
-		class ChildSchema extends builder.object("child-item", {
-			childField: [builder.boolean, builder.handle, builder.string],
-			childData: builder.optional(builder.string),
+		class BroccoliSchema extends builder.object("broccoli-object-schema", {
```

Comment:
I think we need to simplify these tests. I think they're too big for developers to reason about. I would recommend creating a few different _focused_ unit tests. E.g. 1 for just a leaf node tree, 1 for an array, 1 for a map, 1 for an object, and maybe 1 test with a layer or 2 of nesting as a "complex" test - but even this test should be as small as possible so developers can reason about how to fix it if it ever breaks.

Reply:
I think these tests are still much more complex than they need to be. There is a lot of code in each of them, which will make maintenance difficult, and will make it hard for developers to understand what's going on in them.

I would recommend tests as simple as (using map node as an example):
```typescript
const schema = schemaFactory.Map(schemaFactory.number);
const tree = new Map<number>([["foo", 1], ["bar", 2]]);
```
We may want to have 1 test with some schema nesting, but probably just 1, and we should strive to keep that one as simple as possible too.

Also, be sure to name the tests after what scenario they're testing. E.g. `SharedTree - leaf node`.